Peer.Connectors/GithubPullRequestSource.cs

```diff
@@ -14,15 +14,15 @@ public class GithubPullRequestSource : IPullRequestSource<GithubPullRequestSourc
         private static readonly Regex _paramRegex = new Regex(@"(?<orgName>[\w\-]+)/(?<repoName>[\w\-]+)/issues/\d+");
         private readonly AppConfig _config;
         private static IClient<UserGithubClient> _userClient;
-        private static GitHubClient _githubClient;
+        private static IGitHubClient _githubClient;
```

Comment:
Why static?

Reply:
The `IClient` is static in the `AzdoPullRequestSource` too so whatever the conclusion is here probably applies there too.

Reply:
and `GitlabPullRequestSource`